App/src/main/webapp/script.js

```diff
@@ -0,0 +1,26 @@
+function state(){
+    const state = 'Just califonia for now.';
```

Comment:
Should be California instead of califonia

Reply:
Thanks I didn't catch that I'll update now